tests/test_rxname.py

```diff
@@ -24,9 +24,9 @@ def _hash(s):
     return h
 
 def test_hash():
-    rxs = ["a", "hello", "total", "wowza", "z_2na", "z_3na", "absorption", "np", 
-           "n2a", "z_2n2a", "nd", "nt", "nHe3", "nd3a", "nt2a", "z_4n", 
-           "fission_fourth", "z_2np", "z_3np", "n2p", "npa", "n_0", "n_1", "n_2", 
+    rxs = ["a", "hello", "total", "wowza", "z_2na", "z_3na", "absorption", "np",
```

Comment:
I've never seen this line before, "hello" and "wowza"?

Reply:
Not going to hold it up for this, just thought it was interesting!

Reply:
haha yeah, it is just there to test the hash function